.github/workflows/test.yml

```
@@ -0,0 +1,33 @@
+name: test
+
+on:
+  workflow_call:
+
+jobs:
+  test-affected-libs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Setup Environment
+        uses: ./.github/actions/setup-environment
+
+      - name: Run affected Tests
+        run: |
+          npx nx affected -t test --watch=false --browsers=ChromeHeadless --coverage
+
+      - name: Find affected paths
+        uses: ./.github/actions/pathfinder-lcov-normalizer
+
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: package
```

```suggestion
          name: coverage-reports
```
Would something like this make sense for naming?